index.bs

```diff
@@ -2758,44 +2793,69 @@ Appeal by Advisory Committee Representatives</h3>
+	on the proposal to overturn the decision.
+
+	The conclusion of the [=Advisory Committee Override=] is final:
+	the same decision cannot be appealed more than once.
+
+	If the [=Advisory Committee Override=]
+	approves the proposal to overturn the decision,
+	the proposal <em class=rfc2119>may</em> be returned for additional work,
+	after which the applicable decision process is re-initiated.
+
+<h3 id="ac-override">
+Advisory Committee Override</h3>
+
+	An <dfn export>Advisory Committee Override</dfn> is an exceptional two-step procedure
+	used to resolve certain matters where the usual [=consensus=] process
+	and its [[#addressing-fo|escalation path]] are insufficient.
+
+	Note: Currently, this only applies to [=AC Appeals=]
+	and [=recalls=] of [=AB=] or [=TAG=] participants.
+
+	First, the [=Team=] <em class="rfc2119">must</em> announce the proposal for a [=Advisory Committee Override=]
 	to the [=Advisory Committee=]
 	and provide a mechanism for [=Advisory Committee representatives=]
-	to respond with a statement of positive support for this appeal.
+	to respond with a statement of positive support for holding a vote.
 	The archive of these statements <em class="rfc2119">must</em> be [=member-only=].
+
 	If, within <span class="time-interval">one week</span> of the Team's announcement,
-	5% or more of the [=Advisory Committee=] support the appeal request,
-	the Team <em class="rfc2119">must</em> organize an appeal vote
+	5% or more of the [=Advisory Committee=] support holding the vote,
+	the Team <em class="rfc2119">must</em> organize a vote
 	asking the [=Advisory Committee=]
-	“Do you approve of the Decision?”
-	together with links to the decision and the appeal support.
+	“Do you approve of the Advisory Committee Override proposal?”
+	together with details about the proposal
+	and links to support for holding the vote.
 
 	The ballot <em class="rfc2119">must</em> allow for three possible responses:
 	“Approve”,
 	“Reject”,
 	and “Abstain”,
 	together with Comments.
 
-	If the number of votes to reject
-	exceeds the number of votes to approve,
-	the decision is overturned.
-	In that case, there are the following possible next steps:
-
-	<ol>
-		<li>
-			The proposal is rejected.
-
-		<li>
-			The proposal is returned for additional work,
-			after which the applicable decision process is re-initiated.
-	</ol>
+	The level of support needed for a [=Advisory Committee Override=] to pass
+	depends on the level of participation by [=Advisory Committee Representatives=],
+	(including those who cast an explicit “abstain” ballot):
+	* if no more than 15%,
+		the vote passes
+		if the number of “Approve” ballots exceeds three times the number of “Reject” ballots.
+	* if more than 15% but less than 20%,
+		the vote passes
+		if the number of “Approve” ballots exceeds twice the number of “Reject” ballots.
+	* if 20% or more,
+		the vote passes
+		if the number of “Approve” ballots exceeds the number of “Reject” ballots.
```

Comment:
Also possible... and `more than 15% but fewer than 20%` could become `between 15% and 20%` --
```suggestion
	(including those who cast an explicit “abstain” ballot). The vote passes in any of the following cases:
	* if 15% or fewer participate,
	  and the number of “Approve” ballots exceeds triple (3x) the number of “Reject” ballots
	* if more than 15% but fewer than 20% participate,
	  and the number of “Approve” ballots exceeds double (2x) the number of “Reject” ballots
	* if 20% or more participate,
	  and the number of “Approve” ballots exceeds the number of “Reject” ballots
```